api/app/responses/api.py

```diff
@@ -192,23 +191,22 @@ def send_confirmation(self, user, response):
             LOGGER.error('Could not connect to the database to retrieve response confirmation email data on response with ID : {response_id}'.format(response_id=response.id))
 
         try:
-            subject = 'Your application to {}'.format(event.get_description('en'))
             question_answer_summary = strings.build_response_email_body(answers)
 
-            template = email_repository.get(event.id, 'confirmation-response').template
-            body_text = template.format(
-                title=user.user_title,
-                firstname=user.firstname,
-                lastname=user.lastname,
-                event_description=event.get_description('en'),
-                question_answer_summary=question_answer_summary,
-                event_name=event.get_name('en'))
-            emailer.send_mail(
-                user.email, 
-                subject, 
-                body_text=body_text, 
-                sender_name=g.organisation.name,
-                sender_email=g.organisation.email_from)
```

Comment:
Seems like the `sender_name` and `sender_email` aren't being set anymore. I think this is on of the only usages though from all the other emails so probably best for consistency.

Reply:
Yeah, the send_mail function sets sender_name and sender_email to these values anyway, so the previous code was redundant :)